torch/csrc/utils/schema_info.cpp

```diff
@@ -67,10 +67,34 @@ bool SchemaInfo::is_mutable(size_t index) {
   if (!alias_maps_current_) {
     generateAliasMaps();
   }
+
+  static const c10::FunctionSchema batch_norm_schema = torch::jit::parseSchema(
+      "aten::batch_norm(Tensor input, Tensor? weight, Tensor? bias, Tensor? running_mean, Tensor? running_var, bool training, float momentum, float eps, bool cudnn_enabled) -> Tensor");
+  static const c10::FunctionSchema instance_norm_schema = torch::jit::parseSchema(
+      "aten::instance_norm(Tensor input, Tensor? weight, Tensor? bias, Tensor? running_mean, Tensor? running_var, bool use_input_stats, float momentum, float eps, bool cudnn_enabled) -> Tensor");
+
+  // Note that the batch norm and instance norm checks depend on index because
+  // of cases where either running_mean or running_var alias another input
+  // argument causing its alias status to change.
   return std::any_of(
       input_alias_map_[index].begin(),
       input_alias_map_[index].end(),
-      [this](size_t index) { return this->schema_.is_mutable(index); });
+      [this](size_t index) {
```

Comment:
sorry, one more request I didn't notice last time - could you rename this from `index` to something that doesn't shadow a previous variable? (actually I got confused in the last review and that's why I had the previous suggestion).